FreeRTOS/FreeRTOS-Plus-TCP
# Update Renesas RX network interface port

Description
-----------
This PR updates the network interface for the Renesas RX devices (CC RX compiler), with respect to the multi network endpoint , IPv6 related FreeRTOS+TCP stack changes.

Test Steps
-----------
Tested the following using E2 studio [FreeRTOS+TCP demo for RX72N Envision Kit](https://github.com/tony-josi-aws/Renesas_RX72N_Envision_Kit_CC_FreeRTOS_TCP) project:

- DHCP
- IPv4 pings
- RA/ND [IPv6]
- DNSv4
    - `dns query4: 'aws.amazon.com' = 99.86.12.77 rc = 0`
- DNSv6
    - `dns query6: 'google.nl' = 2404:6800:4007:81d::2003 rc = 0`
- IPv6 pings (ICMPv6)

[Sample test results.](https://github.com/tony-josi-aws/Renesas_RX72N_Envision_Kit_CC_FreeRTOS_TCP/blob/master/readme.txt)

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
